apps/docs/src/app/core/component-docs/icon-tab-bar/examples/icon-tab-bar-example.class.ts

```diff
@@ -0,0 +1,96 @@
+import { SemanticColor, TabConfig } from '../../../../../../../../libs/core/src/lib/icon-tab-bar/types';
+import { Directive, Input, OnInit } from '@angular/core';
+
+@Directive()
+export abstract class IconTabBarExampleClass implements OnInit {
+
+    @Input()
+    itemsWithSubItems = 0;
+
+    @Input()
+    counter = false;
+
+    @Input()
+    hasIcon = true;
+
+    // max = 18; If you want set more, pls provide more icons.
+    @Input()
+    tabsLength = 10;
+
+    @Input()
+    subItemsLevel = 1;
+
+    items: TabConfig[] = [];
+
+    private icons = [
```

Comment:
private property should be decorated with @hidden, and start with _ prefix

Reply:
If this is an example - we don't need to add typedoc comments.